### DISCUSSION TITLE
blog/istio-servicemesh-api-gateway/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#330] [Announcements]
# blog/istio-servicemesh-api-gateway/

服务网格与 API 网关之间是什么关系？是不是使用了 Istio 就可以替代了 API 网关？Istio 的 API 网关是如何运作的？有哪些方式暴露 Istio mesh 中的服务？这篇文章给为你解答。

https://jimmysong.io/blog/istio-servicemesh-api-gateway/

**Comment by lai3d** [comment]
学了会儿Kong API Gateway, 然后一堆问题就冒出来，正好是群主写的这些
